### DISCUSSION TITLE
css background-image not showing

### DISCUSSION THREAD

**Discussion by gith0r** [community/community#117192] [Programming Help] [answered]
### Body

hey everyone, i have an issue on my website which is really weird, now when i lunch it using a local host from vs code, everything is fine and all the imgs are shown weather in html or css, but when i lunched it using github and made it live, i ran into an issue, the background images in css are not shown and the error in the console is " Failed to load resource: the server responded with a status of 404 ()"  now the idea is when hover over a button it shows the img as it supposed too, but when i do it shows this error and i noticed when in the application section in the console when i hover over the button it creates a file with the same name and formate as the imgs that suppose to be shown and it says nothing to preview as shown in this screenshot:
![image](https://github.com/community/community/assets/81085842/b8e60077-c359-4bef-9727-5ffb58c01205)

and you can see there is two files of theTrilogy.jpg, one of them is already there and shows the img fine and the other is created when hover over the button and says nothing to perview, please help.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by HeyFang** [comment]
are all images not visible or just the one with .jpg suffix?
if so then try converting them into png format

**Reply by gith0r** [reply]
nope, i checked and png and jpg are not showing

**Comment by MarouaneBenbetka** [comment]
hello @gith0r check the images paths , maybe you're using an image path that only work in local

**Reply by gith0r** [reply]
well as i checked on the internet, i followed the same structure for the pathing, and i tried different things i added a double full stops and what not but still not working, if you know how can i use a global path that would be awesome.

**Reply by MarouaneBenbetka** [reply]
can you please send me the image paths you're using .

**Reply by gith0r** [reply]
![image](https://github.com/community/community/assets/81085842/5d8e5342-a488-4fa1-8f8d-c4622d432dc5)
well as you can see here i tried two different methods and they both fails, but as i said before they work just fine on the local host( when i lunch the site from visual studio)

**Comment by MarouaneBenbetka** [comment, marked as answer]
hello again @gith0r  , 
the problem is here the path needs to be `/theWeeknd/imgs/theTrilogy.jpg`
cause when hosting it starts from  the root is  `https://gith0r.github.io` and not  `https://gith0r.github.io/theWeeknd`

if this fixed you issue please mark as answered , good luck khalil !

**Reply by gith0r** [reply]
i tried it but unfortunately its still the same issue

**Reply by MarouaneBenbetka** [reply]
no man it worked, check the website , maybe you didn't wait for it to get deployed. 
try to delete the cache .

**Reply by gith0r** [reply]
lmao it is working, yeah my bad thanks alot for the help dude

**Reply by MarouaneBenbetka** [reply]
Your welcome man, happy that it worked.
